src/components/Monitoring/dashboard/SeedBankDashboard.tsx

```diff
@@ -64,6 +68,15 @@ export default function SeedBankDashboard(props: SeedBankDashboardProps): JSX.El
   const [defaultSensorTimePeriod, setDefaultSensorTimePeriod] = useState<string>();
   const [defaultPvTimePeriod, setDefaultPvTimePeriod] = useState<string>();
   const [defaultSensor, setDefaultSensor] = useState<Device>();
+  const timeZoneFeatureEnabled = isEnabled('Timezones');
+  const tz = useLocationTimeZone().get(seedBank);
+
+  const onChangeTimeZone = (newTimeZone: TimeZoneDescription | undefined) => {
+    updatePreferences({
+      ...monitoringPreferences,
+      timeZone: newTimeZone?.id,
```

Comment:
i don't know if we need to update the user preference here, might be good to always use seed bank time zone on load. Otherwise we have to give additional info and allow user to reset back to seed bank time zone.

Reply:
maybe we need a reset button always?

Reply:
Ok, I will not save it in preferences for now